thinkhazard/scripts/s3helper.py

```diff
@@ -39,3 +43,24 @@ def download_file(self, object_name, file_name=None):
             logging.error(e)
             return False
         return True
+
+    def delete_object(self, object_name):
+        try:
+            self.s3_client.delete_object(
+                Bucket=self.bucket,
+                Key=object_name
+            )
+        except ClientError as e:
+            logging.error(e)
+            return False
+        return True
+
+    def check_object_exists(self, object_name):
```

Comment:
@arnaud-morvan Should this be implemented differently to not always provoke an exception in tween if maintenance file is not present in s3?

Reply:
Seems good, if there is no more reliable way to get the info.